tjreports/media/js/tjrContentUI.js

```diff
@@ -51,8 +51,15 @@ jQuery.extend(tjrContentUI.report, {
 		if (!doProcess) {
 			return false;
 		}
-		jQuery(".hasPopover").popover('destroy')
-		var promise = tjrContentService.postData(this.url+'&tpl='+layout, this.$form.serialize());//, {'datatype':'html'}
+
+		try {
```

Comment:
document this somewhere, also add comment why if else

Reply:
added to doc